### ISSUE TITLE
index.d.ts

### ISSUE THREAD

**Issue by justincheung0** [RobLoach/node-raylib#93]
So when you use dts-gen it does not read the c++ and header files, therefore, making all function definitions null, meaning all functions besides a few exceptions have no arguments so what do I do?

**Comment by Googcoon** [comment]
yeah the index.d.ts file provided has a lot of functionality included but is missing a few functions here and there
